ui/page/show/view.jsx

```diff
@@ -159,7 +159,7 @@ export default function ShowPage(props: Props) {
       const windowHref = window.location.href;
       const noUrlParams = search.length === 0;
       if (windowHref.includes('?') && noUrlParams) {
-        history.replaceState(history.state, '', windowHref.substring(0, windowHref.length - 1));
+        history.replaceState(history.state, '', windowHref.substring(0, windowHref.length));
```

Comment:
If `substring`-ing the entire string, then can just use the original.
But I think the code is trying to remove a trailing `?`, which the new code wouldn't achieve.  The second parameter is the exclude index, which has become out of bounds.